serverless-stepf.yml

```diff
@@ -1,5 +1,4 @@
 rBuilds:
-  Comment: "R Builds"
```

Comment:
@stevenolen hates comments?

Reply:
oh, they took this out of the spec, i had forgotten i made this change!